crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -808,13 +808,6 @@ define_tedge_config! {
     },
 
     apt: {
-        /// The filtering criterion that is used to filter packages list output by name
-        #[tedge_config(example = "tedge.*")]
-        name: String,
-        /// The filtering criterion that is used to filter packages list output by maintainer
-        #[tedge_config(example = "thin-edge.io team.*")]
-        maintainer: String,
-
```

Comment:
As noted by @reubenmiller:

> The old apt.name and apt.maintainer (inclusive) filters still need to be accepted

For that purpose, the `tedge_config` macro features `deprecated_key`s.

See for example, how [`mqtt.external.port` has been deprecated in favor of `mqtt.external.bind.port`](https://github.com/thin-edge/thin-edge.io/blob/main/crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs#L622)